watertap/unit_models/gac.py

```diff
@@ -1058,52 +1058,78 @@ def initialize_build(
 
     # ---------------------------------------------------------------------
 
-    # TODO: def _get_performance_contents(self, time_point=0):
-    # TODO: def _get_stream_table_contents(self, time_point=0):
+    def _get_performance_contents(self, time_point=0):
+        var_dict = {}
+        expr_dict = {}
+
+        var_dict["temp"] = self.bed_volume
+
+    # ---------------------------------------------------------------------
+    def _get_stream_table_contents(self, time_point=0):
+        return create_stream_table_dataframe(
+            {
+                "Process Inlet": self.inlet,
+                "Process Outlet": self.outlet,
+                "Adsorbed Contaminant Outlet": self.adsorbed,
+            },
+            time_point=time_point,
+        )
 
     # ---------------------------------------------------------------------
     def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
 
-        # overwrite default scaling
+        # scale based on molar flow traditionally provided by user for building flowsheets
```

Comment:
The only thing I have left is handling multi-component property packs and I can consider this when making the edits. I still have the consideration that for GAC it is easy to define the saturation capacity of the particle and adsorption rate of the single solute. When multiple solutes are present, you can still reasonably determine the saturation capacity of the particle and adsorption rate of the target solute, where the background solutes reduce both values. The decision is to then treat the backgrounds as inert where in = out with reductions to saturation capacity of the particle and adsorption rate, or to also determine the saturation capacity of the particle and adsorption rate of the background solute(s) which gets a little more complicated. These background solutes would then also selectively adsorb into the particle and be contained in the waste stream.

Reply:
That sounds fine, but my main point was that-- if I were to pass in even a single ion via the ion prop model (or IDAES generic prop model), though it wouldn't make sense in terms of electroneutrality, that ion would not be captured by `self.config.property_package.solute_set`. That is because when a component is defined as either Cation or Anion type, which would happen when a non-zero charge is provided for a component, that component would be listed under `self.config.property_package.ion_set`. Neutral components (zero charge) are listed in `self.config.property_package.solute_set`.

Reply:
Yes, I gotcha. If treating all others as inert, it may even be,

`target_component = "TCE"`
`inert_set = self.config.property_package.component_list.remove(target_component)`

Depends on how indexing and scaling is easiest to implement wrt the sets.